### PR TITLE
Events api

### PR DESCRIPTION
**Pull request by lbliii**
events api (no use cases since getting deprecated?)

### CODE REVIEW

File: content/api/api-events.md [hunk 1]
```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
...
-      }
-    ],
-    "type": "INFO",
-    "data": {
-      "elementId": "us-east-1:i-123a4567",
-      "level": "WARNING",
-      "message": "WARNING: Event is triggering!"
+
+## About the Events API
+
```
Comment (rclagett) [inline content/api/api-events.md]: The about section is missing.

File: content/api/api-events.md [hunk 2]
```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
...
+| eventTags | query | string | You can tag events to enable filtering in CloudWisdom by passing in keys and values. The format for tags is [{“name”:”key1″, “value”:”value1″}, {“name”:”key2″, “value”:”value2″}, (etc.)] |
+| policy | query | Array[string] | Policy Id. |
+| isExternal | query | boolean |  |
+| title | query | Array[string] | Title of the event |
+| type | query | Array[string] | Defines type of event; valid inputs are INFO, WARNING, or CRITICAL. |
+
+### Request URL
+
+```
+https://app.metricly.com/events?startTime={startTime}&endTime={endTime}&elementId={elementId}
```
Comment (rclagett) [inline content/api/api-events.md]: Could we use the copyable code span for request URLs like in the other new API docs?

File: content/api/api-events.md [hunk 3]
```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
...
+
+| Parameter | Parameter Type | Data Type | Description |
+|--------------------|----------------|-----------|----------------------------------------------|
+| policyId | path | string | A policy's unique ID. |
+
+
+### Request URL
+
+```
+https://app.metricly.com/events/delete/policy/{policyId}
```
Comment (rclagett) [inline content/api/api-events.md]: Could we use the copyable code span for request URLs like in the other new API docs?

File: content/api/api-events.md [hunk 4]
```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
...
-| tags | Optional | You can tag events to enable filtering in CloudWisdom by passing in keys and values. The format for tags is [{“name”:”key1″, “value”:”value1″}, {“name”:”key2″, “value”:”value2″}, (etc.)] |
-| data | Required | The data attribute provides CloudWisdom with the content for the actual event. It contains three attributes: **elementId (required)**: This should contain the fully qualified name (fqn) of the element you wish to generate an external event for. **level (optional)**: The severity level of the event. Valid inputs are Info, Warning, or Critical. If values are set at both level and type, the level value will take precedence since it’s closer to the message itself and more specific. **message (optional)**: The message text generated for the event. |
-|source |optional | Source of the event. This is set to the ingested integration if the value is missing. |
-| title| required | Title of the event |
-|timestamp   | optional  | You can define a timestamp, but it must be in the epoch format. If you do not define a timestamp, CloudWisdom will create one for you.  |
+```
+https://app.metricly.com/events/{eventId}
```
Comment (rclagett) [inline content/api/api-events.md]: Could we use the copyable code span for request URLs like in the other new API docs?

File: content/api/api-events.md [hunk 5]
```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
...
+| startTime | query | date-time | Start time for the query. YYYY-MM-DDT00:001Z format (must include time zone). |
+| endTime | query | date-time | End time for the query. YYYY-MM-DDT00:001Z format (must include time zone). |
+| category | query | Array[string] | Defines event category; valid inputs are INFO, WARNING, or CRITICAL. |
+| elementId | query | Array[string] | Element Ids can be found in by selecting an element in CloudWisdom and viewing the URL (/#/inventory/6bdf4fd1-1111-1111-1c11-17eb5f628e46) |
+| elementName | query | Array[string] | Name of element. |
+| elementType | query | Array[string] | Type of element; examples include EC2, ALB, SERVER. |
+| elementTags | query | string | Tags associated to an element. The format for tags is [{“name”:”key1″, “value”:”value1″}, {“name”:”key2″, “value”:”value2″}, (etc.)] |
+| eventTags | query | string | You can tag events to enable filtering in CloudWisdom by passing in keys and values. The format for tags is [{“name”:”key1″, “value”:”value1″}, {“name”:”key2″, “value”:”value2″}, (etc.)] |
+| policy | query | Array[string] | Policy Id. |
+| isExternal | query | boolean |  |
```
Comment (rclagett) [inline content/api/api-events.md]: isExternal description is missing. It's whether or not the event is an external event.

File: content/api/api-events.md (hunk 2 above)
Comment (rclagett) [inline content/api/api-events.md]: The URL should be just /events. The optional params _can_ be used which would make the URL different but the starting point is just /events

```suggestion
https://app.metricly.com/events
```

File: content/api/api-events.md [hunk 6]
```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
...
+## About the Events API
+
+
+## GET from /events
+
+{{< button href="https://app.metricly.com/swagger-ui.html#!/events/getEventsUsingGET_1" theme="info" >}} GET {{< /button >}} Use this endpoint to get details on events and their associated policies/elements.
+
+{{% expand "View Method Details." %}}
+
+### Parameters
```
Comment (rclagett) [inline content/api/api-events.md]: These parameters are optional as a user could just make a request to /events and get events for the past hour. We should make a note about that.

File: content/api/api-events.md [hunk 7]
```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
...
+```
+https://app.metricly.com/events/delete/policy/{policyId}
 ```
-This method utilizes the Ingest API endpoint.
 
-Replace {apiId} in the above URL with the API key from your Custom integration. To access this key, click Integrations in the user account drop-down menu in the bottom left-hand corner of CloudWisdom, then find the CUSTOM integration and copy the API key from the API Key column.
+### CURL
 
-**Parameters**
+In the following CURL example, events for a policy with the **policyId** `e321f5fa-a6fb-49d6-b4e6-beb468eb461f` are deleted.
```
Comment (rclagett) [inline content/api/api-events.md]: Does the policy ID need to be copyable here? It's unlikely a user would have to copy it given they have their own IDs.

File: content/api/api-events.md [hunk 8]
```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
...
-| title| required | Title of the event |
-|timestamp   | optional  | You can define a timestamp, but it must be in the epoch format. If you do not define a timestamp, CloudWisdom will create one for you.  |
+```
+https://app.metricly.com/events/{eventId}
+
+```
+
+### CURL
+
+In the following CURL example, the **eventId** `82691285-5254-4830-ba07-34b809cfe25b` is used.
```
Comment (rclagett) [inline content/api/api-events.md]: I don't think the event ID needs to be copyable either. Though, I do like them having a different format. Is there a way to make it a code span without the copy button?

Comment (lbliii) [reply content/api/api-events.md]: I'm unsure if there's another way to do codespan without it auto-generating the copy button. I like to format the value in code but yeah this happens haha.

File: content/api/api-events.md (hunk 7 above)
Comment (lbliii) [reply content/api/api-events.md]: it's more for the code font style, the copy thing is just auto generated

**Comment by lbliii** [comment]
@CryptoPishka  I believe this is ready for re-review. Updated based on ryan's feedback

File: content/api/api-events.md [hunk 9]
```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
...
 ---
-The events API allows you to consume events generated by CloudWisdom. You can either grab a list of events generated by CloudWisdom or grab an individual event by its ID. To explore CloudWisdom’s REST API and test some requests, go here.
 
-**Request Header**
+## About the Events API
 
-| Header Name | Header Value |
-|--------------|------------------|
-| Content-Type | application/json |
+The Events API can be used to view and delete events generated by policy alerts in CloudWisdom. Responses include details about elements and policies associated to the events found.
```
Comment (rclagett) [inline content/api/api-events.md]:
```suggestion
The Events API can be used to view and delete events generated by policy alerts in CloudWisdom. Responses include details about elements and policies associated to the events found. You can test these endpoints by visiting our [Swagger page](https://app.metricly.com/swagger-ui.html#/elements) and by clicking the interactive buttons below.
```

File: content/api/api-events.md [hunk 10]
```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
...
+      "eventTags": {}
     }
+  ]
+}
+```
+{{% /expand %}}
+
+---
+
+## DELETE from /events/delete/policy/{policyId}
```
Comment (rclagett) [inline content/api/api-events.md]:
```suggestion
## DELETE to /events/delete/policy/{policyId}
```

**Pull request merged**